### PR TITLE
Created Prodsec Blog Part

### PR DESCRIPTION
**Pull request by atharv-g-kulkarni**

### CODE REVIEW

File: vue-security-page/index.html [hunk 1]
```diff
@@ -9,6 +9,10 @@
       href="https://fonts.googleapis.com/css2?family=Red+Hat+Text:ital,wght@0,300..700;1,300..700&display=swap"
       rel="stylesheet"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Red+Hat+Display:ital,wght@0,300..900;1,300..900&display=swap"
+      rel="stylesheet"
+    />
```
Comment (hybridx) [inline vue-security-page/index.html]: Import this in main styles

**Review by hybridx — CHANGES_REQUESTED** [review]

**Comment by hybridx** [comment]
rest LGTM

File: vue-security-page/index.html (hunk 1 above)
Comment (atharv-g-kulkarni) [reply vue-security-page/index.html]: removed from index.html and added to main.css

**Review by yashoswalyo — APPROVED** [review]
LGTM

**Review by yashoswalyo — APPROVED** [review]
LGTM

**Comment by yashoswalyo** [comment]
Can you move `.github` folder out of `vue-security-page` in root

File: vue-security-page/src/components/blog/ProdsecBlog.vue [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            models and weakness patterns to design and build with security as a
+            primary objective:
+          </p>
+          <ul class="sdl-links-list">
+            <li><a>Software supply chain security assurance</a></li>
+            <li><a>Security testing process</a></li>
+          </ul>
+        </div>
+        <div class="pf-v5-l-grid__item pf-m-0-col pf-m-2-col-on-lg"></div>
+        <div
```
Comment (rohit-bharmal) [inline vue-security-page/src/components/blog/ProdsecBlog.vue:29]: Can we use pf-card here for all of this cards ?

File: vue-security-page/src/components/blog/ProdsecBlog.vue [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  </div>
+</template>
+
+<script setup></script>
+
+<style scoped>
+.security-overview > div {
+  padding-top: 48px;
+}
+.security-overview li {
```
Comment (rohit-bharmal) [inline vue-security-page/src/components/blog/ProdsecBlog.vue:141]: add one line space after one css property completed.

**Comment by hybridx** [comment]
Please use eslint

**Comment by hybridx** [comment]
https://dev.to/ankitt8/setting-up-eslint-airbnb-configuration-prettier-and-husky-pre-commit-hooks-in-cra-5dbo

**Review by hybridx — APPROVED** [review]

**Pull request merged**